lib/endpoints/class-wp-json-users-controller.php

```diff
@@ -208,6 +208,9 @@ public function create_item( $request ) {
 			}
 			$user->ID = $user_id;
 		}
+
+		$this->update_additional_fields_for_object( $user, $request );
```

Comment:
If an `update_option` or `update_post_meta` fails (DB is too busy, perhaps), it's important that we pass this data back up the entire stack. I'm fine with us treating each field as atomic by itself though, rather than the whole set of fields as atomic (i.e. one failure doesn't break immediately).